cmds/info.c

```diff
@@ -999,6 +1037,12 @@ void process_uftrace_info(struct uftrace_data *handle, struct uftrace_opts *opts
 	else
 		process(data, "# %-20s: %s", "recorded on", ctime(&statbuf.st_mtime));
 
+#if 0
+	/* This is better not to be printed. */
+	if (info_mask & UTC_OFFSET)
+		process(data, fmt, "utc offset", info->utc_offset);
```

Comment:
I think the `info->utc_offset` is for internal information so better not to be printed in `uftrace info` output.

Reply:
It's now removed after another update.